Makefile

```diff
@@ -12,22 +12,37 @@ goerli:
 # mainnet:
 # 	make all environment=mainnet
 
-all: download_artifacts generate build cmd-help
+local:
```

Comment:
When we run Geth on our local we use the "development" name. But here we use "local" name to fetch the artifacts for the "development" env and in the logs I see: 
```
Environment          local
```
even though the installation scripts run for "development". Do you think we can stick to "development" name in this case?

Reply:
Network and environment names got confusing. We've got following networks in hardhat:
- `hardhat` (default) - local hardhat node
- `development` - local geth node
- `goerli` - public test network
- `mainnet` - public main network

To use contracts packages as dependencies during the development, in the CI we build artifacts and publish them to NPM tagged `development`. These artifacts can be used as compiled artifacts but we cannot use the addresses to call them, as they were deployed to internal CI's hardhat node during the CI workflow execution.

In the client, we also pull those packages tagged `development` to build the client with the latest version of contract bindings.

An alternative path is when a developer wants to deploy contracts on their own machine and use the artifacts in the client. The network in hardhat they use is `development`, to not confuse it with the package the client already expects, I used the name `local`.

I'm open to straighten it up. But since renaming will have quite significant impact I'd like to have a broader discussion.

CC: @michalinacienciala @pdyraga

Reply:
I think it makes sense to have a `development` name for local geth node. It also was fine for the packages to be named `development` pre local geth node time. But yea, now it's a bit confusing.

I would rename `development` npm packages to ex. `integration` since they run on CI and drop the `development` package name. However, I'm not sure how much time and effort it takes to do it.

Maybe we can start smaller, and change the logs from `Environment: local -> development` and add the explanation in the comments?

Reply:
We could rename the `development` network in the `hardhat.config.ts` in all projects to `local`. The package we publish to the NPM Registry will still be tagged `development`.
WDYT @michalinacienciala?

But let's not block this PR on that.

Reply:
7f95f56d62f8d48c67f698124d0a106b0350dc59

Reply:
> We could rename the development network in the hardhat.config.ts in all projects to local. The package we publish to the NPM Registry will still be tagged development.
WDYT @michalinacienciala?
Sounds good to me, I prefer this from changing the names of the packages.

Reply:
https://github.com/keep-network/keep-core/issues/3157